### PR TITLE
Add a fix to make OpenJDK works with setcap on java 

### PR DESCRIPTION
**Pull request by allamand**
- This fix may be removed once https://github.com/docker-library/official-images/pull/6028 will be merged

- Also rewrite Dockerfile for better readiness

Signed-off-by: sebastien allamand <sebastien.allamand@orange.com>

### CODE REVIEW

**Review by jal06 — APPROVED** [review]
LGTM

**Comment by cscetbon** [comment]
@allamand time to revert it. The branch has been merged

File: Dockerfile [hunk 1]
```diff
@@ -53,53 +52,59 @@ ENV \
...
+    if [ -n "$CQLSH_CONTAINER" ]; then apt-get -y --no-install-recommends install python; else rm -rf  $CASSANDRA_HOME/pylib; fi; \
+    apt-get update && apt-get -qq -y install \
        libcap2-bin \
        procps \
-       dnsutils \
-    && apt-get -y purge wget jq localepurge \
-    && apt-get -y autoremove \
-    && apt-get -y clean \
-    && rm -rf \
+       dnsutils; \
```
Comment (cscetbon) [inline Dockerfile:103]: @allamand Why replacing `&&` by `;` ??? We want it to fail if any statement fails.

Comment (allamand) [reply Dockerfile:103]: even with ; it will can fail :)

**Pull request merged**